packages/website/src/theme/MDXComponents/RuleExtensionNotice.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from '@docusaurus/Link';
+import React from 'react';
+
+import { Feature } from './Feature';
+
+export function RuleExtensionNotice(): React.JSX.Element {
+  return (
+    <Feature emoji="🧱">
+      This is an "extension" rule that replaces a core ESLint rule to work with
+      TypeScript. See{' '}
+      <Link href="/rules#extension-rules">Rules &gt; Extension Rules</Link> for
+      more information.
+    </Feature>
+  );
+}
```

Comment:
Is this used anywhere? I couldn't find any usage of this, and it hasn't been added to [`src/themes/MDXComponents/index.tsx`](https://github.com/typescript-eslint/typescript-eslint/blob/844608bf1f14b43c734dd6a3f0bcd9ef9a51b25b/packages/website/src/theme/MDXComponents/index.tsx#L10-L18) too.

My knowledge of MDX is limited, basing this off https://docusaurus.io/docs/markdown-features/react#mdx-component-scope, so this might be an oversight be me.

Reply:
Aha you're totally right, I'd split the notice out from `RuleAttributes` and then undid the change only in that file. Dead code. Nice.